labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -98,9 +85,6 @@ object ListManipulationExercise01 {
    *
    * Implement it whatever way suites you best. Hint: it can be done in a neat way using recursion. 
    */
-  def tails[T](l: List[T]): List[List[T]] = {
-    error("fix me")
-  }
+  def tails[T](l: List[T]): List[List[T]] = if (l.isEmpty) List(l) else List.concat(List(l), tails(l.tail))
+  
```

Comment:
For adding a single element to the beginning of a list, it’s usually faster (both coding time and runtime) to use cons (the `::` operator) instead of `concat`.